sdk/tables/azure-data-tables/src/table_clients.cpp

```diff
@@ -569,10 +570,10 @@ Azure::Response<Models::DeleteEntityResult> TableClient::DeleteEntity(
     Core::Context const& context)
 {
   auto url = m_url;
-  url.AppendPath(
+  url.AppendPath(Azure::Core::Url::Encode(
       m_tableName + PartitionKeyFragment
       + Azure::Core::Url::Encode(tableEntity.GetPartitionKey().Value) + RowKeyFragment
-      + Azure::Core::Url::Encode(tableEntity.GetRowKey().Value) + ClosingFragment);
+      + Azure::Core::Url::Encode(tableEntity.GetRowKey().Value) + ClosingFragment));
```

Comment:
Do we need to encode this?

If we are encoding the closing fragment of `)`, we might need to encode Row and partition key fragment as well (contains `=` and the partition key fragment starts with `(`),

But I suspect this is building up a set of query parameters, so none of those need to be encoded, but I am not sure. What's the expected format sent to the service?

It seems to be something like the following in the url path: `...TableName(PartitionKey='SomeValue',RowKey='AnotherValue')...`

If we really need to encode this, turn the constant strings into pre-encoded values to avoid having to encode the same thing over and over.